client/containers/DeleteEventForm.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { RelatedPlannings } from '../components'
+import { connect } from 'react-redux'
+import { reduxForm } from 'redux-form'
+import * as actions from '../actions'
+
+export class Component extends React.Component {
+
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        const { error, handleSubmit, initialValues, handlePlanningClick } = this.props
+        return (
+            <form onSubmit={handleSubmit}>
+                {error && <div className="error-block">{error}</div>}
+                <label>{initialValues.name}</label>
+                {initialValues._plannings && initialValues._plannings.length > 0 &&
+                    <div>
+                        <br/>Related Plannings for this event&nbsp;<br/>
+                        <RelatedPlannings plannings={initialValues._plannings} onPlanningClick={handlePlanningClick}/>
+                    </div>
+                }
+                <button type="submit" style={{ visibility: 'hidden' }}>Submit</button>
+            </form>
+        )
+    }
+}
+
+Component.propTypes = {
+    handleSubmit: React.PropTypes.func.isRequired,
+    handlePlanningClick: React.PropTypes.func.isRequired,
+    initialValues: React.PropTypes.object.isRequired,
+    error: React.PropTypes.object,
+}
+
+// Decorate the form component
+export const FormComponent = reduxForm({
```

Comment:
For another ticket I made a Confirmation modal component here: https://github.com/superdesk/superdesk-planning/pull/138

You could use it :)